src/seedu/addressbook/data/person/Address.java

```diff
@@ -2,18 +2,32 @@
 
 import seedu.addressbook.data.exception.IllegalValueException;
 
+
 /**
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String EXAMPLE = "123, some street, #56-78, 091011";
+    public static final String ADDRESS_FORMAT = "BLOCK, STREET, UNIT, POSTAL CODE";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses must be in format " + ADDRESS_FORMAT;
+    public static final String ADDRESS_VALIDATION_REGEX = ".+,.+,.+,.+";
+
+    private static final String ADDRESS_SPLIT_REGEX = ",";
+    private static final Integer BLOCK_INDEX = 0;
+    private static final Integer STREET_INDEX = 1;
+    private static final Integer UNIT_INDEX = 2;
+    private static final Integer POSTALCODE_INDEX = 3;
+
+    //public static final String ADDRESS_VALIDATION_REGEX = ".+";
```

Comment:
Commented codes shouldn't be inside the commit; you have revision control to keep past important codes.